src/pyroboplan/visualization/set_get_joints.py

```diff
@@ -0,0 +1,98 @@
+from pyroboplan.core.utils import (
```

Comment:
not sure if `visualization` is the right module for this, it's probably good under `src/pyroboplan/core` as a new file there.

Also the name `set_get_joints.py` maybe should be replaced with `named_joint_configurations.py` or something like that?